tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -554,14 +544,14 @@ module.exports = {
       // addresses users and groups
       selector: '.files-collaborators-collaborator'
     },
-    collaboratorInformationByCollaboratorName: {
-      selector: '//*[contains(@class, "files-collaborators-collaborator-name") and .="%s"]/ancestor::*[contains(concat(" ", @class, " "), " files-collaborators-collaborator ")]',
-      locateStrategy: 'xpath'
-    },
     collaboratorInformationSubName: {
       // within collaboratorsInformation
       selector: '.files-collaborators-collaborator-name'
     },
+    collaboratorInformationByCollaboratorName: {
+      selector: '//*[contains(@class, "files-collaborators-collaborator-name") and .="%s"]/ancestor::*[contains(concat(" ", @class, " "), " files-collaborators-collaborator ")]',
+      locateStrategy: 'xpath'
+    },
```

Comment:
remove this if you're using `collaboratorsDialog().elements.collaboratorInformationByCollaboratorName.selector` above

Reply:
not all the methods use `collaboratorsDialog().elements.collaboratorInformationByCollaboratorName.selector` like method `getCollaboratorsList` so this must be in this pageObject as well in collaboratorsDialog.js

Reply:
In that case we need to change the methods that don't use it. the whole point of page objects is to have the selectors in one place, in case they change they should only be changed in a single place

Reply:
It has been fixed in PR https://github.com/owncloud/phoenix/pull/3095